docs/docs/administration/backup-and-restore.md

```diff
@@ -45,7 +45,7 @@ docker compose up -d    # Start remainder of Immich apps
   <TabItem value="Windows system (PowerShell)" label="Windows system (PowerShell)">
 
 ```powershell title='Backup'
-docker exec -t immich_postgres pg_dumpall --clean --if-exists --username=postgres > "\path\to\backup\dump.sql"
+docker exec -t immich_postgres pg_dumpall --clean --if-exists --username=postgres | Set-Content -Encoding utf8 "\path\to\backup\dump.sql"
```

Comment:
Can we add `C:\path\to\backup\dump.sql` to replicate what we have in the restore instructions?